lib/time/time.Time.js

```diff
@@ -361,7 +361,13 @@
 	};
 
 	function pad( number, digits ) {
-		return ( 1e12 + Math.abs( number ) + '' ).slice( -digits );
+		number = '' + Math.abs( number );
```

Comment:
No and no :) We have no consensus on that, so -1 wouldn't be appropriate. Having it in our code style guide would make sense, though.